src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/iOS/SwitchExtensions.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Linq;
+using UIKit;
+
+namespace Xamarin.Platform
+{
+	public static class SwitchExtensions
+	{
+		public static void UpdateIsToggled(this UISwitch uiSwitch, ISwitch view)
+		{
+			uiSwitch.SetState(view.IsToggled, true);
+		}
+
+		public static void UpdateTrackColor(this UISwitch uiSwitch, ISwitch view) =>
+			uiSwitch.UpdateTrackColor(view, UISwitch.Appearance.OnTintColor, uiSwitch.GetOffTrackColor());
+
+		public static void UpdateTrackColor(this UISwitch uiSwitch, ISwitch view, UIColor? defaultOnTrackColor, UIColor? defaultOffTrackColor)
+		{
+			if (view == null)
+				return;
+
+			if (view.TrackColor == Forms.Color.Default)
+				uiSwitch.OnTintColor = defaultOnTrackColor;
+			else
+				uiSwitch.OnTintColor = view.TrackColor.ToNative();
+
+			UIView uIView;
+			if (NativeVersion.IsAtLeast(13))
+				uIView = uiSwitch.Subviews[0].Subviews[0];
+			else
+				uIView = uiSwitch.Subviews[0].Subviews[0].Subviews[0];
```

Comment:
Could we create a` UIView FindChildren <T>` or similar extension method? It would be useful in other cases, and does this type of case where we need to access a specific child easier.

Reply:
This feels fairly edge case since I'm basically just accessing a specific child and I'm not "Searching" for a specific type